openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -118,30 +135,36 @@ components:
           $ref: '#/components/schemas/ErrorReport'
 
     AzureLandingZone:
-      description: Created Azure Landing Zone details
+      description: Created Azure Landing Zone details.
       type: object
       properties:
         id:
-          description: An identifier of created Azure landing zone
+          description: An identifier of created Azure landing zone.
           type: string
         resources:
-          description: List of Azure landing zone deployed resources
+          description: List of Azure landing zone deployed resources.
           type: array
           items:
             $ref: '#/components/schemas/AzureLandingZoneDeployedResource'
 
     AzureLandingZoneDeployedResource:
-      description: Details of an Azure resource
+      description: Details of an Azure resource.
       type: object
       properties:
         resourceId:
-          description: Unique Azure resource identifier
+          description: Unique Azure resource identifier.
           type: string
         resourceType:
-          description: Azure resource type
+          description: Azure resource type.

```

Comment:
What values might this have? Could it include a link to where they are defined?

Reply:
The ARM resource type for deployed resources: https://docs.microsoft.com/en-us/azure/templates/
The resource types are located under the Reference node.

Reply:
f.ex. "type": "Microsoft.Compute/virtualMachines" or "type": "Microsoft.Network/virtualNetworks/subnets"

Reply:
Would you add that to the description? It would help the naive reader (e.g., me :))